spec/acceptance/badge_email_spec.rb

```diff
@@ -11,7 +11,7 @@
   scenario "Send new badge notification email" do
     expect{BadgeMailer.new_badge_notification(
       badge.id, user.id
-    ).deliver}.to change{ActionMailer::Base.deliveries.count}.by(1)
+    ).deliver_now}.to change{ActionMailer::Base.deliveries.count}.by(1)
```

Comment:
Expression at 14, 18 should be on its own line.<br>Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.